providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/resolver/process/storage/FlagStore.java

```diff
@@ -132,4 +135,27 @@ private void streamerListener(final Connector connector) throws InterruptedExcep
         log.info("Shutting down store stream listener");
     }
 
+    private List<String> getChangedFlagsKeys(Map<String, FeatureFlag> newFlags) {
+        Map<String, FeatureFlag> changedFlags = new HashMap<>();
```

Comment:
We can already work with lists here; maybe even one Set is enough to which we add those elements directly.